nucypher/blockchain/eth/utils.py

```diff
@@ -85,18 +85,19 @@ def estimate_block_number_for_period(period: int, seconds_per_period: int,  late
 def etherscan_url(item, network: str, is_token=False) -> str:
     if network is None or network is UNKNOWN_DEVELOPMENT_CHAIN_ID:
         raise ValueError("A network must be provided")
-    elif network == 'mainnet':
+
+    network = network.lower()
+    if network == 'mainnet':
```

Comment:
Should we use the `NetworksInventory.MAINNET` constant?

Reply:
`NetworksInventory` refers to NuCypher networks, and it's just a coincidence that Ethereum and NuCypher's mainnets are both called "mainnet".

Reply:
Ah! 😞